bidsify/bidsify.py

```diff
@@ -89,16 +88,16 @@ def bidsify_workflow(dicom_dir, heuristics, subject, session=None, output_dir='.
             wk_file.write('.heudiconv/\ntmp/\nvalidator.txt\n')
 
     # Run heudiconv
-    cmd = f'heudiconv {dir_type} {dicom_dir} \
-            -s {subject} -f {heuristics} -c dcm2niix \
-            -o {output_dir} --bids --overwrite --minmeta'
+    cmd = (f'heudiconv {dir_type} {dicom_dir}'
```

Comment:
f the whole thing (outside the parens)

Reply:
That doesn't work in any IDE I've tried it in.

Reply:
well darn. nvm then it's all good.